clang/test/lit.cfg.py

```diff
@@ -332,43 +332,6 @@ def calculate_arch_features(arch_string):
     config.available_features.add("llvm-driver")
 
 
-def exclude_unsupported_files_for_aix(dirname):
-    for filename in os.listdir(dirname):
-        source_path = os.path.join(dirname, filename)
-        if os.path.isdir(source_path):
-            continue
-        f = open(source_path, "r", encoding="ISO-8859-1")
-        try:
-            data = f.read()
-            # 64-bit object files are not supported on AIX, so exclude the tests.
-            if (
-                any(
-                    option in data
-                    for option in (
-                        "-emit-obj",
-                        "-fmodule-format=obj",
-                        "-fintegrated-as",
-                    )
-                )
-                and "64" in config.target_triple
-            ):
-                config.excludes += [filename]
-        finally:
-            f.close()
-
-
-if "aix" in config.target_triple:
-    for directory in (
-        "/CodeGenCXX",
-        "/Misc",
-        "/Modules",
-        "/PCH",
-        "/Driver",
-        "/ASTMerge/anonymous-fields",
-        "/ASTMerge/injected-class-name-decl",
```

Comment:
Just to confirm, so this removal does not have any impact on clang tests? I think some PCH/Modules cases would have some expected failures in AIX object mode, like `Objective-C support is unimplemented`?

Reply:
There were no failures from running check-all. I also ran the tests in PCH/Modules individually and they pass.

Reply:
OK